projects/arlas-toolkit/src/lib/tools/utils.ts

```diff
@@ -324,3 +331,15 @@ export function hasContributorData(contributor: Contributor): boolean {
     return false;
   }
 }
+
+export function orderAlphabeticallyArlasSearchFields(arlasSearchFieldA: ArlasSearchField, arlasSearchFieldB: ArlasSearchField,){
+  const startLetter = arlasSearchFieldA?.label.trim()[0];
+  const startLetterCmp = arlasSearchFieldB?.label.trim()[0];
+  if(startLetter > startLetterCmp) {
```

Comment:
To properly sort, there should be a comparison that goes further than just the first caracter, otherwise it would be a weird order. It could be done recursively for this if/else section

Reply:
i ll update